keras_cv/models/__init__.py

```diff
@@ -166,5 +178,8 @@
     YOLOV8Detector,
 )
 from keras_cv.models.segmentation import DeepLabV3Plus
+from keras_cv.models.segmentation import MaskDecoder
```

Comment:
May be better as SAMMaskDecoder for clarity

Reply:
Done.